packages/manager/apps/web/client/app/hosting/cdn/shared/hosting-cdn-shared.service.js

```diff
@@ -0,0 +1,313 @@
+import { HOSTING_CDN_SHARED_SETTINGS_DOMAIN_ENABLE, HOSTING_CDN_SHARED_SETTINGS_DOMAIN_DISABLE } from './hosting-cdn-shared.constants';
+
+export default class HostingCdnSharedService {
+
+  /* @ngInject */
+  constructor($http, OvhHttp) {
+    this.$http = $http;
+    this.OvhHttp = OvhHttp;
```

Comment:
I suggest you to just use `$http` provider. And replace all the `OvhHttp[METHOD]` to `$http[METHOD]`